bin/cli.js

```diff
@@ -148,7 +149,24 @@ const setRequestHeaders = () => {
   addUserAgentHeader('HubSpot CLI', pkg.version);
 };
 
+const NO_CONFIG_VALIDATION = {
+  init: { skip: true },
+  auth: { skip: true },
+};
+
```

Comment:
Nit: we could compress this to:
```typescript
const NO_CONFIG_VALIDATION = {
  init: true,
  auth: true,
};
```

And simplify the check below from: 
`command => !(NO_CONFIG_VALIDATION[command] && NO_CONFIG_VALIDATION[command].skip)` 

to 

`command => !NO_CONFIG_VALIDATION[command]`

Reply:
I'm changing this up in the next PR (https://github.com/HubSpot/hubspot-cli/pull/1310), so going to leave for now.